From fb914c20a21f62ba9a1e1f3c718bf84046bf0fe0 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 26 Jan 2012 18:11:10 -0500
Subject: [PATCH] build: Add --disable-documentation option

Useful for operating system builders who want to avoid cyclic build
dependencies.
---
 configure.ac     |    5 +++++
 help/Makefile.am |    6 +++++-
 2 files changed, 10 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9b10a70..b5be442 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,6 +22,11 @@ AC_SUBST(LIB_PANEL_APPLET_LT_VERSION)
 
 GNOME_MAINTAINER_MODE_DEFINES
 
+AC_ARG_ENABLE([documentation],
+	AS_HELP_STRING([--enable-documentation], [enable man pages and HTML]),
+	[], [enable_documentation=yes])
+AM_CONDITIONAL(ENABLE_DOCUMENTATION, test x$enable_documentation = xyes)
+
 LT_PREREQ([2.2.6])
 LT_INIT([dlopen disable-static])
 
diff --git a/help/Makefile.am b/help/Makefile.am
index 7d38ddd..688b08e 100644
--- a/help/Makefile.am
+++ b/help/Makefile.am
@@ -1,3 +1,7 @@
-SUBDIRS = clock fish
+DIST_SUBDIRS = clock fish
+SUBDIRS =
+if ENABLE_DOCUMENTATION
+SUBDIRS += $(DIST_SUBDIRS)
+endif	
 
 -include $(top_srcdir)/git.mk
-- 
1.7.6.5

